WordPressKit/WordPressAPIError.swift

```diff
@@ -54,3 +67,7 @@ extension WordPressAPIError: LocalizedError {
     }
 
 }
+
+protocol HTTPURLResponseProviding {
+    var httpResponse: HTTPURLResponse? { get }
```

Comment:
I want to avoid potential conflicts with other variable named `response`, considering it's a pretty generic name. But no, I haven't thought about including "URL" in the name. "HTTP response" seems pretty nature. 😄